### DISCUSSION TITLE
Recommendations for Argo Rollouts on Multi-cluster Architecture.

### DISCUSSION THREAD

**Discussion by Cstack-Krish** [argoproj/argo-rollouts#3579] [General]
We have two AKS clusters, one in the primary region and another in its paired region (an active-active configuration), the applications in both clusters are the same. We are using argo rollouts to use canary deployment strategy. Both the clusters have its own argocd installed with the argo-rollouts. Is it possible for the argo in paired region cluster to sync along with the changes made in the rollouts in the primary cluster? For example if I perform a Promote-Full rollout action in the primary cluster's argo, the same action on the same application should happen on paired region's cluster.  Are there any solutions to the same?

**Comment by kostis-codefresh** [comment]
There is nothing like this out of the box.

You either need to use the CI system to coordinate everything or have the first rollout use notifications to notify the second one.

More discussion

- https://github.com/argoproj/argo-rollouts/issues/2737
- https://github.com/argoproj/argo-rollouts/issues/2088
